src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesVsPatterns.cs

```diff
@@ -174,7 +174,10 @@ void Test(object? x)
             c.VerifyDiagnostics(
                 // (7,18): error CS0150: A constant value is expected
                 //         if (x is nonConstant)
-                Diagnostic(ErrorCode.ERR_ConstantExpected, "nonConstant").WithLocation(7, 18)
+                Diagnostic(ErrorCode.ERR_ConstantExpected, "nonConstant").WithLocation(7, 18),
+                // (9,13): warning CS8602: Dereference of a possibly null reference.
```

Comment:
>warning CS8602: Dereference of a possibly null reference. [](start = 27, length = 57)

can we avoid this cascading diagnostic?

Reply:
Not really.  For all we know the constant is `null`.  The test gives no information about the flow state.

---
In reply to: [381010560](https://github.com/dotnet/roslyn/pull/41503#discussion_r381010560) [](ancestors = 381010560)